content/id/docs/tasks/tools/kubeadm/high-availability.md

```diff
@@ -0,0 +1,364 @@
+
+## Tugas-tugas umum setelah _bootstrapping_ _control plane_
+
+### Menginstal _worker_
+
+Node _worker_ bisa digabungkan ke klaster menggunakan perintah yang kamu simpan sebelumnya
+dari keluaran perintah `kubeadm init`:
+
+```sh
+sudo kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866
+```
+
+## Distribusi sertifikat manual {#manual-certs}
+
+Jika kamu memilih untuk tidak menggunakan `kubeadm init` dengan opsi `--upload-certs` berarti kamu harus 
+menyalin sertifikat dari Node _control plane_ utama secara manual ke
+Node _control plane_ yang akan bergabung.
+
+Ada beberapa cara untuk melakukan hal ini. Pada contoh berikut ini kami menggunakan `ssh` dan `scp`:
+
+SSH dibutuhkan jika kamu ingin mengendalikan seluruh Node dari satu mesin.
+
+1.  Nyalakan ssh-agent pada perangkat utamamu yang memiliki akses ke seluruh Node pada
+    sistem:
+
+    ```
+    eval $(ssh-agent)
+    ```
+
+2.  Tambahkan identitas SSH milikmu ke dalam sesi:
+
+    ```
+    ssh-add ~/.ssh/path_to_private_key
+    ```
+
+3.  Lakukan SSH secara bergantian ke setiap Node untuk memastikan koneksi bekerja dengan baik.
+
+    - Ketika kamu melakukan SSH ke Node, pastikan untuk menambahkan opsi `-A`:
+
+        ```
+        ssh -A 10.0.0.7
+        ```
+
+    - Jika kamu menggunakan sudo pada Node, pastikan kamu menyimpan _environment_ yang ada sehingga penerusan SSH
+      dapat bekerja dengan baik:
+
+        ```
+        sudo -E -s
+        ```
+
+4. Setelah mengatur SSH pada seluruh Node kamu harus menjalankan skrip berikut pada Node _control plane_ pertama setelah
+   menjalankan `kubeadm init`. Skrip ini akan menyalin sertifikat dari Node _control plane_ pertama ke Node
+   _control plane_ lainnya:
+
+    Pada contoh berikut, ganti `CONTROL_PLANE_IPS` dengan alamat IP dari
+    Node _control plane_ lainnya.
+    ```sh
+    USER=ubuntu # dapat disesuaikan
+    CONTROL_PLANE_IPS="10.0.0.7 10.0.0.8"
+    for host in ${CONTROL_PLANE_IPS}; do
+        scp /etc/kubernetes/pki/ca.crt "${USER}"@$host:
+        scp /etc/kubernetes/pki/ca.key "${USER}"@$host:
+        scp /etc/kubernetes/pki/sa.key "${USER}"@$host:
+        scp /etc/kubernetes/pki/sa.pub "${USER}"@$host:
+        scp /etc/kubernetes/pki/front-proxy-ca.crt "${USER}"@$host:
+        scp /etc/kubernetes/pki/front-proxy-ca.key "${USER}"@$host:
+        scp /etc/kubernetes/pki/etcd/ca.crt "${USER}"@$host:etcd-ca.crt
+        # Kutip baris berikut jika kamu menggunakan etcd eksternal
+        scp /etc/kubernetes/pki/etcd/ca.key "${USER}"@$host:etcd-ca.key
+    done
+    ```
+
+    {{< caution >}}
+    Salinlah hanya sertifikat yang berada pada daftar di atas saja. Perkakas kubeadm akan mengambil alih pembuatan sertifikat lainnya
+    dengan SANs yang dibutuhkan untuk Node _control-plane_ yang akan bergabung. Jika kamu menyalin seluruh sertifikat tanpa sengaja,
```

Comment:
nit:
```suggestion
    dengan SANs yang dibutuhkan untuk Node _control plane_ yang akan bergabung. Jika kamu menyalin seluruh sertifikat tanpa sengaja,
```

Reply:
Thanks, removed all hyphens for control plane.